app/src/main/java/org/systers/mentorship/view/activities/SettingsActivity.kt

```diff
@@ -1,8 +1,14 @@
 package org.systers.mentorship.view.activities
 
 import android.content.Intent
+import android.graphics.Color
+import android.net.Uri
+import android.os.Build
 import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
+import android.support.v4.media.session.PlaybackStateCompat
+import android.support.v7.app.AlertDialog
+import android.widget.Toast
```

Comment:
are you using this? if not remove please

Reply:
import android.support.v7.app.AlertDialog 
This is being used . I will remove the others .